### PR TITLE
Split brain clustercheck WIP

### PR DESCRIPTION
**Pull request by TomasHradecky**
splitbrain condition through xinetd and clustercheck script [WIP]

### CODE REVIEW

**Comment by cloudlab-autobuild** [comment]
Can one of the admins verify this patch?

File: galera/slave.sls [hunk 1]
```diff
@@ -187,4 +187,66 @@ galera_service:
   - reload: true
 
 {%- endif %}
+
```
Comment (epcim) [inline galera/slave.sls]: do not edit this file - I will remove

**Comment by epcim** [comment]
also add's additional user setup, same way we introduced in: https://github.com/salt-formulas/salt-formula-mysql/pull/9

**Comment by pupapaik** [comment]
Please wait until my verification before we merge this change.

File: tests/integration/slave_cluster/checks_clustercheck_spec.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+describe file('/etc/xinetd.d/mysql_clustercheck_9200') do
```
Comment (epcim) [inline tests/integration/slave_cluster/checks_clustercheck_spec.rb]: clustercheck.conf

File: tests/integration/master_cluster/checks_clustercheck_spec.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+describe file('/etc/xinetd.d/mysql_clustercheck.conf') do
+    it('should exist')
+    its('content') { should match /clustercheck password clustercheck 1 1/ }
```
Comment (epcim) [inline tests/integration/master_cluster/checks_clustercheck_spec.rb]: I was wrong here... (trap for @TomasHradecky) we should match "clustercheck clustercheck password" the user/password are switched

**Review by epcim — CHANGES_REQUESTED** [review]
fix minors, extend test... I will test it before the merge.
actual merge is not a problem, as not enabled by default, but we have to keep an eye before we merge the counterpart patch to reclass system-level.

**Comment by epcim** [comment]
closing in favor of squashed PR #30

**Pull request closed without merging**